library/src/main/scala/scala/scalajs/js/Dynamic.scala

```diff
@@ -70,6 +70,9 @@ sealed trait Dynamic extends js.Any with scala.Dynamic {
 
   @JSOperator def &&(that: js.Dynamic): js.Dynamic = js.native
   @JSOperator def ||(that: js.Dynamic): js.Dynamic = js.native
+
+  /** <span class="badge badge-ecma2016" style="float: right;">ECMAScript 2016</span> */
+  @JSOperator def **(that: js.Dynamic): js.Dynamic = js.native
```

Comment:
I think it is the right thing to do this, but we need to remember that this is a backwards source incompatible change for the release notes.

Reply:
Adding a method to a public API is always backward source incompatible in Scala, since there can be extension methods (through implicit classes in Scala 2). That said, we can call out this one in particular, sure.